packages/forms-web-app/src/views/examination/about-the-application.njk

```diff
@@ -26,18 +26,22 @@
             <div class="govuk-grid-column-two-thirds">
               <p class="results">
                 <strong>
-                  <span id="results-number">{{ noOfProjects }}</span>
-                  {% if typeList.length === 0 %}
-                    No documents found. Please try with different search text.
+                  {% if documents %}
+                    <span id="results-number">{{ pageData.totalItems }}</span>
+                    {% if pageData.totalItems === 1 %}
+                      result
+                     {% else %}
+                      results
+                     {% endif %}
                   {% else %}
-                    <span id="results-number">{{ pageData.totalItems }}</span> results
+                    No documents found. Please try with different search text.
                   {% endif %}
                 </strong>
               </p>
             </div>
 
             <button id="toggleFilters" class="govuk-button govuk-button">Show search and filters</button>
-            <form action="/document-library/search/{{ caseRef }}/1" method="POST" novalidate>
+            <form action="/examination/about-the-application/{{ caseRef }}/search/1" method="POST" novalidate>
```

Comment:
Should this link be hard-coded?  Looks like it's now out of step with the route change?

Reply:
all routes/links updated to have /examination/{{case_ref}}